### PR TITLE
feat: add support for ubuntu 22.04

### PR DESCRIPTION
**Pull request by leoalves100**
# Description

- Add support for Ubuntu 22.04

## Issues Resolved

List any existing issues this PR resolves

## Check List

- [x] A summary of changes made is included in the CHANGELOG under `## Unreleased`
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by ramereth** [comment]
@leoalves100 can you please resolve the rspec and cookstyle failures?

**Comment by leoalves100** [comment]
@ramereth adjustment done

**Comment by damacus** [comment]
Looks like there's some fun with the matrix to fix yet

https://github.com/sous-chefs/mysql/actions/runs/3740217446/jobs/6348344596

**Comment by RenatoMorenoR** [comment]
Hi friends, how can we help with this PR 😄. We have several system with this cookbook and we started migration to ubuntu 22.04 and we encountered the problem that it is not compatible. I am available to be able to help in whatever way we can to push this development. Thanks 😄

**Comment by damacus** [comment]
@RenatoMorenoR thanks for the help. We do need the test passing before we can merge this in. 

I've updated some of the testing in #697. So this will also need rebasing.

**Comment by leoalves100** [comment]
@damacus Thank you for solving the tests. I can't identify why [check-metadata](https://github.com/sous-chefs/mysql/actions/runs/4663761772/jobs/8255383650?pr=685) is failing, can you help me please?

**Comment by DeWaRs1206** [comment]
@leoalves100 @damacus any chance to have this PR reviewed ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,9 +4,7 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
-## 11.0.15 - *2023-07-10*
-
-## 11.0.14 - *2023-05-17*
+- feat: add support for ubuntu 22.04
```
Comment (ramereth) [inline CHANGELOG.md:7]: Please move this under ``Unreleased`` and don't remove the other lines you removed.

**Review by ramereth — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (leoalves100) [reply CHANGELOG.md:7]: @ramereth The changelog file is correct, can you merge this PR please?

**Comment by kitchen-porter** [comment]
Released as: [11.1.0](https://github.com/sous-chefs/mysql/releases/tag/11.1.0)

**Pull request merged**